django/db/backends/oracle/operations.py

```diff
@@ -54,6 +54,8 @@ class DatabaseOperations(BaseDatabaseOperations):
     cast_data_types = {
         'TextField': cast_char_field_without_max_length,
     }
+    # We do not currently support explaining query execution plans on Oracle
+    explain_prefix = None
```

Comment:
In `base/operations.py` the comment says "Empty string if the backend does not support this.". Here you've got `None`. 

Given `explain_prefix` will be `''` unless overridden, do we need anything at all here?

Reply:
Nope, I will remove this! Good catch